x-pack/plugins/enterprise_search/server/routes/app_search/curations.ts

```diff
@@ -115,4 +115,21 @@ export function registerCurationsRoutes({
       path: '/as/engines/:engineName/curations/find_or_create',
     })
   );
+
+  router.get(
+    {
+      path: '/api/app_search/engines/{engineName}/curation_search',
```

Comment:
I have no idea if this is a good API route name, sorry y'all 😬 Def feel free to give suggestions, I was worried `curations/search` made it seem like you're searching _for_ curations rather than searching for documents 😕